front-end/src/pages/AuthorPage.tsx

```diff
@@ -99,31 +120,31 @@ export default function AuthorPage(props: any) {
 
 
   const displayProfileButtons = () => {
-    if (props.loggedInUser && author?.id === props.loggedInUser.authorId) {
-      return <ListGroupItem tag="a" href={`/author/${props.loggedInUser.authorId}/followers`}>Follows</ListGroupItem>
-    } else {
+    if (author?.id !== props.loggedInUser.authorId) {
```

Comment:
what if `props.loggedInUser` is undefined?

Reply:
good catch, I'm like 50% sure the if statement fails and then returns nothing (see line 147)

Reply:
if `props.loggedInUser` is undefined then IIRC we might be at risk of an undefined error such as `Error: Cannot read property 'authorId' of undefined`. If we can get typing on the incoming props then TS should be able to point these cases out to us for free

Reply:
fixed